### PR TITLE
Project 4: add "join table" for visits

### PR DESCRIPTION
**Pull request by sarazaToussaint**
## In this pull request I: 

- Use different types of relationships between tables.
- Prepare complex queries that answer analytical questions.
- Use primary key & foreign key mechanisms for joining tables.

     ### Queries' Screenshots

1. Who was the last animal seen by William Tatcher?
Day4One

2. How many different animals did Stephanie Mendez see?
Day4Two

3. List all vets and their specialties, including vets with no specialties.
Day4Three

4. List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
Day4Four

5. What animal has the most visits to vets?
Day4Five

6. Who was Maisy Smith's first visit?
Day4Six

7. Details for most recent visit: animal information, vet information, and date of visit.
Day4Seven

8. How many visits were with a vet that did not specialize in that animal's species?
Day4Eight

9. What specialty should Maisy Smith consider getting? Look for the species she gets the most.
Day4Nine

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -44,3 +44,25 @@ UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon','Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon','Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander','Squirtle','Blossom');
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon','Boarmon');
```
Comment (Mre55) [inline data.sql]: This `UPDATE` are changing the table so from line 38 - 46 should have to be moved in to the `schema.sql` file. Kindly, move those to keep the file structure professional.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @sarazaToussaint,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- All queries are written and result as expected ✅
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (sarazaToussaint) [reply data.sql]: Hi @Mre55, I made the changes, thank you for the code review.

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @sarazaToussaint      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sarazaToussaint** [comment]
Hi @KayLemba, thank you for the code review.

**Pull request merged**